docs/configuration.md

```diff
@@ -0,0 +1,96 @@
+<!--
+    Copyright 2019 VMware, Inc.
+    SPDX-License-Identifier: BSD-2-Clause
+-->
+
+# Precaution configuration
+
+Precaution is using a variety of tools under the hood without exposing
+implementation details and complexity of the underlying tools to the user.
+
+Concretely, this means that Precaution ignores any tool specific configuration
+files which may exist in the project's repository and performs scans based only
+on Precaution specific configuration.
+
+Precaution provides you with the option to use optional `.precaution.json` file.
+This file **must** be in the root directory of your project.
+
+All paths in the options are processed as if they are relative to the root directory.
```

Comment:
Isn't this the same as saying the paths should be absolute rather than relative?  Using the word relative to explain an absolute path might trip up some folks.

Reply:
Yea, maybe you are right. I will change it.